Sources/FoundationEssentials/String/String+Path.swift

```diff
@@ -440,18 +472,16 @@ extension String {
     
     // From swift-corelibs-foundation's NSTemporaryDirectory. Internal for now, pending a better public API.
     internal static var temporaryDirectoryPath: String {
-#if os(Windows)
-        let validPathSeps: [Character] = ["\\", "/"]
-#else
-        let validPathSeps: [Character] = ["/"]
-#endif
-        
         func normalizedPath(with path: String) -> String {
-            if validPathSeps.contains(where: { path.hasSuffix(String($0)) }) {
+            guard path.utf8.last != ._slash else {
+                return path
+            }
+            #if os(Windows)
+            guard path.utf8.last != ._backslash else {
                 return path
-            } else {
-                return path + String(validPathSeps.last!)
             }
+            #endif
+            return path + "/"
```

Comment:
Should this be `path._standardizingSlashes() + "/"`?

Reply:
That seems like a good addition - today on Windows this only gets called with the result of `standardizingPath` which will already have its slashes standardized (so the `if` statement above seems like it might always be a no-op) but adding the extra check here is probably a good measure for future-proofing this code